FrontPage/Client.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+import Apollo
+import AppAuth
+
+let config = Config()
```

Comment:
I noticed that the `config` object is initialised here but is also referenced in `AuthViewController`. I think this could cause problems in the future (for example, if the `AuthViewController` is initialised before the `Client` class). Generally speaking, an object should be initialised and used in the same class.

In this case, given that there is only one `Config` object will be ever needed, and it should only needed be loaded once, I suggest that we add a singleton instance for `Config`, and use that singleton object where it's needed.